src/theme/palette.ts

```diff
@@ -6,24 +6,163 @@
 import { alpha, PaletteOptions } from '@mui/material';
 
 // Palette and PalleteOptions types can be extended here as needed
-/*
+
 declare module '@mui/material/styles/createPalette' {
   interface Palette {
+    buy: PaletteColor;
+    sell: PaletteColor;
   }
 
   interface PaletteOptions {
+    buy: PaletteColorOptions;
+    sell: PaletteColorOptions;
   }
 }
-*/
 
 // TODO: Define MUI palettes for light theme
 export const light: PaletteOptions = {
   mode: 'light',
   divider: alpha('#383838', 0.2), // Todo: Import base color (not alpha) from tokens file (Grey 900)
+  common: {
+    black: '#000',
+    white: '#FFF',
+  },
+  primary: {
+    main: '#A9ABD1',
+    dark: '#5F618D',
+    light: '#DBDDFF',
+  },
+  secondary: {
+    main: '#2D96FF',
+    dark: '#005195',
+    light: '#76C6FF',
+  },
+  error: {
+    main: '#FF274B',
+    dark: '#B60017',
+    light: '#FF6877',
+  },
+  warning: {
+    main: '#FF8D00',
+    dark: '#EA4F01',
+    light: '#FFBE45',
+  },
+  success: {
+    main: '#01C38D',
+    dark: '#01603D',
+    light: '#5DF7BD',
+  },
+  buy: {
+    main: '#2D96FF',
+    dark: '#005195',
+    light: '#76C6FF',
+  },
+  sell: {
+    main: '#FF274B',
+    dark: '#B60017',
+    light: '#FF6877',
+  },
+  grey: {
+    50: '#F9F9F9',
+    100: '#F2F2F2',
+    200: '#E4E4E4',
+    300: '#CFCFCF',
+    400: '#BBBBBB',
+    500: '#7F7F7F',
+    600: '#6B6B6B',
+    700: '#555555',
+    800: '#4D4D4D',
+    900: '#383838',
+    A100: '#F2F2F2',
+    A200: '#E4E4E4',
+    A400: '#BBBBBB',
+    A700: '#555555',
+  },
```

Comment:
Some of these color values, especially colors under `grey` and `common` will be the same between the two themes. They only need to be defined once, and then they can be applied to both themes.